.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+updates:
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: daily
+    time: "04:00"
+  open-pull-requests-limit: 10
+  target-branch: develop
+  ignore:
+  - dependency-name: "@types/node"
+    versions: "^17.0.0"
```

Comment:
_idk if you oversaw it in the pr_
but would it be better if we just stick to 16.6.0 (= remove the "versions" option)?
since >.6. adds new things. but we can't/should't use them/the typings since our min node is only .6., right?